src/ui/components/Controller.tsx

```diff
@@ -297,15 +297,15 @@ const Controller = () => {
 			<div className="bbgm-container position-relative mt-2 flex-grow-1 d-flex flex-column">
 				<Header />
 				<SideBar pageID={pageID} />
-				<div
+				<main
 					className="p402_premium flex-grow-1 d-flex flex-column"
 					id="actual-content"
 				>
 					<div id="actual-actual-content" className="clearfix flex-shrink-0">
```

Comment:
Another bonus of this - you can also get rid of all these flex classes on lines 301 and 304, since `main` only has one child they are no longer doing anything :)